src/dependencies.jl

```diff
@@ -1005,7 +1005,7 @@ macro load_dependencies(args...)
         end
         s = Symbol(sym)
         errorcase = Expr(:block)
-        push!(errorcase.args,:(error("Could not load library "*$(dep.name)*". Try running Pkg.build() to install missing dependencies!")))
+        push!(errorcase.args,:(error("Could not load library "*$(dep.name)*". Try running `Pkg.build(package)` to install missing dependencies!")))
```

Comment:
updated the text here to match the instructions above regarding `Pkg.build(package)`